src/CADL.Extension/Emitter.Csharp/main.cadl

```diff
@@ -0,0 +1,15 @@
+import "@cadl-lang/rest";
```

Comment:
why do we need this file in the root folder?

Reply:
When build, it need to check if the library works or not, so it need a sample cadl file.

Reply:
Remove lint-cadl-library check from build